apps/shipments/iot_client.py

```diff
@@ -36,3 +38,43 @@ def update_shadow(self, device_id, shadow):
             raise AWSIoTError("Error in response from AWS IoT")
 
         return iot_shadow['data']
+
+    def get_list_owner_devices(self, owner_id, active=None, in_bbox=None):
+        LOG.debug(f'Getting devices for: {owner_id} from AWS IoT')
+        log_metric('transmission.info', tags={'method': 'DeviceAWSIoTClient.get_list_owner_devices'})
+
+        if in_bbox:
+            in_bbox = ','.join([c.strip() for c in in_bbox.split(',')])
+
+        loop = True
+        next_token = None
+        results = []
+        while loop:
+            params_dict = dict(ownerId=owner_id,
+                               active=active if active is not None else '',
+                               in_bbox=in_bbox if in_bbox else '',
```

Comment:
`in_bbox if in_bbox else ''` - could you combine this logic with line 46?
`in_bbox = ','.join([c.strip() for c in in_bbox.split(',')]) if in_bbox else ''`